src/main/resources/application.yaml

```diff
@@ -48,6 +48,8 @@ accesstoken:
   serviceConfig:
     - serviceName: sscs
       validity: ${SAS_TOKEN_VALIDITY:300} #In seconds
+    - serviceName: test
```

Comment:
Should this value come from terraform, where test container is defined?

Reply:
Agree, that would be ideal to have it declared only once. Refactoring for another PR?

Reply:
It's actually a tiny change - all you need to add is an output variable that points to your azurerm_storage_container's name. And then in applycation.yaml refet the environment variable that Jenkins built from that output var.

Reply:
If it goes for `test` service, the same should be applied for `sscs` and it becomes refactoring.